shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/metadata/generator/PipelineDDLGenerator.java

```diff
@@ -132,39 +134,38 @@ private Optional<String> decorate(final DatabaseType databaseType, final String
     }
     
     private Collection<String> generateActualDDLSQL(final DatabaseType databaseType, final String schemaName, final String actualTable, final DataSource dataSource) throws SQLException {
-        return CreateTableSQLGeneratorFactory.findInstance(databaseType).orElseThrow(() -> new ShardingSphereException("Failed to get dialect ddl sql generator"))
+        return CreateTableSQLGeneratorFactory.findInstance(databaseType).orElseThrow(() -> new ServiceProviderNotFoundException(DatabaseType.class))
                 .generate(actualTable, schemaName, dataSource);
     }
```

Comment:
`new ServiceProviderNotFoundException(DatabaseType.class))` could be improved, seems `new ServiceProviderNotFoundException(CreateTableSQLGenerator.class, databaseType))` could be used.